tests/upgrade_path/upgrade_helpers.py

```diff
@@ -219,3 +219,48 @@ def upgrade_test_helper(duthost, localhost, ptfhost, from_image, to_image,
 
     if enable_cpa and "warm-reboot" in reboot_type:
         ptfhost.shell('supervisorctl stop ferret')
+
+
+def check_asic_and_db_consistency(pytest_config, duthost, consistency_checker_provider):
+    if not pytest_config.getoption("enable_consistency_checker"):
+        logger.info("Consistency checker is not enabled. Skipping check.")
+        return
```

Comment:
Non-blocking comment: `pytest.skip` can be used to represent that a test has been skipped.

Reply:
We don't want to skip the test in this case, just the consistency check within it.

Reply:
Ah right, I remember the discussion about this now.